### DISCUSSION TITLE
[Keyball61] 表側のランドが剥離してしまいました

### DISCUSSION THREAD

**Discussion by totorono** [Yowkees/keyball#210] [Q&A]
左手トラックボールで制作しました。promicroのピンソケットのはんだ付けをする際、誤って必要以上にはんだを流し込んでしまい、
ピンソケットの穴が塞がってしまいました。このピンソケットを取り除く際にランドが剥離してしまったのですが、大丈夫でしょうか？まだリカバリ可能であればご助力頂きたいです。
剥離したのはトラックボールの無い側で、表面です。よろしくお願いいたします。
![20230329_183850](https://user-images.githubusercontent.com/130044728/230330990-aba8edc2-9244-45e3-a6b7-7602d7de075b.jpg)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。状況了解しました。
剥がれたのは表面だけでしょうか？裏面に半田が乗るようであればそのまま組み立てていただいて大丈夫です。
ただ、剥がれた事によりフルカラーLEDのデータ線が切れましたが、LEDを搭載しないのであれば問題ありません。

もしフルカラーLEDを乗せる予定があるようでしたら、ミドル基板単体での販売も可能です。その場合、お送り先等をご連絡する必要がありますので、白銀ラボウェブサイトの問合せよりメッセージをいただければ幸いです。

以上、ご検討よろしくお願いします。

**Comment by totorono** [comment]
対応していただいてありがとうございます。
一応最後まで組み立てたのですが、親機（USB接続した方）しかキーが反応しませんでした。子機が使えなくなってしまうのでやはり基盤を購入したいです。
